<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clang-format-GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="formatOptionsGroupBox">
       <property name="title">
        <string>Format Options</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="formatOptionsListWidget">
          <item>
           <property name="text">
            <string>Style</string>
           </property>
           <property name="toolTip">
            <string>Set Style options</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tabs</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="detailsGroupBox">
        <property name="title">
         <string>Details</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QStackedWidget" name="detailsStackWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="styleOptionsPage">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>185</width>
               <height>126</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QRadioButton" name="llvmStyleRButton">
                <property name="text">
                 <string>LLVM coding standard</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="googleStyleRButton">
                <property name="text">
                 <string>Google's C++ style guide</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="chromiumStyleRButton">
                <property name="text">
                 <string>Chromium's style guide</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QRadioButton" name="mozillaStyleRButton">
                <property name="text">
                 <string>Mozilla's style guide</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QRadioButton" name="webkitStyleRButton">
                <property name="text">
                 <string>Webkit's style guide</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="TabsOptionsPage">
            <widget class="QWidget" name="layoutWidget_1">
             <property name="geometry">
              <rect>
               <x>1</x>
               <y>0</y>
               <width>391</width>
               <height>53</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="useTabsLbl">
                <property name="text">
                 <string>Use Tabs:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="useTabsComboBox">
                <item>
                 <property name="text">
                  <string>Never</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>For indentation</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Always</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="2">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="tabWidthLbl">
                <property name="text">
                 <string>Tab Width:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="tabWidthSpinBox">
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>16</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="srcPreviewGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Preview</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QToolButton" name="openOriginalSrcToolButton">
           <property name="toolTip">
            <string>Open new source file as the original</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../ResourcesIcons.qrc">
             <normaloff>:/Resources/document-new-6.png</normaloff>:/Resources/document-new-6.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>125</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTabWidget" name="srcPreviewTabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="originalSrcTab">
            <attribute name="title">
             <string>Original</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2"/>
           </widget>
           <widget class="QWidget" name="formattedSrcTab">
            <attribute name="title">
             <string>Formatted</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ResourcesIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>formatOptionsListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>detailsStackWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
